layouts/community/ergodox/germ/readme.md

```diff
@@ -0,0 +1,23 @@
+# Gergo
+
+TODO: Update image
+![Gergo](http://i.imgur.com/q2M3uEU.jpg)
+
+A compact 50% (14x4) Split Keyboard compatible with i2c modules and a trackball. 
+
+[More info on qmk.fm](http://qmk.fm/gergo/)
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+Hardware Supported: Gergo (Kit, Partial, Ready)
+Hardware Availability: [gboards.ca](http://gboards.ca) 
+
+##Firmware building
+After cloning the QMK repo and installing dfu-programmer build with.
+
+    make gergo:germ:dfu
```

Comment:
This actually also flashes the board, if you want to just build it with the germ keymap, it's just

```suggestion
    make gergo:germ
```